stripe/src/main/java/com/stripe/android/view/CountryUtils.kt

```diff
@@ -17,6 +17,11 @@ internal object CountryUtils {
             Country(code, Locale("", code).displayCountry)
         }
 
+    private fun localizedCountries(currentLocale: Locale) =
+        Locale.getISOCountries().map { code ->
+            Country(code, Locale("", code).getDisplayCountry(currentLocale))
+        }
```

Comment:
We should probably always do this instead of accessing `COUNTRIES`

Reply:
Right, can we just use `localizedCountries()` everywhere instead of `COUNTRIES` which is never updated?